NewSpring/ApollosProject
# [DEV-3693] Add Prayer Preview Card

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a prayer preview card and displays it when you tap on either the church, campus, or community tab of the prayer menu.
![Screenshot 2019-04-22 13 30 54](https://user-images.githubusercontent.com/3229463/56514284-dfc3c580-6502-11e9-9eba-ff205fca086a.png)

### How do I test this PR?
1. Pull this branch
2. Start up the simulator
3. Tap on the prayer menu icon
4. Tap on the menu tabs. The prayer preview card should appear when you tap on the church, campus, or community tabs. This is static content and will not change per tab for MVP. 

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] It is a portion of  [DEV-3693](https://newspring.atlassian.net/browse/DEV-3693)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
